example/lib/location_indicator_example_page.dart

```diff
@@ -135,7 +135,8 @@ class _LocationIndicatorExamplePageState
     await Geolocator.requestPermission();
     final location = await Geolocator.getLastKnownPosition();
     if (!mounted || location == null) return;
-
+    // Small delay to let the map render before moving the camera
+    await Future.delayed(Duration(milliseconds: 500));
```

Comment:
Is this just a cosmetic thing so it looks nice or does this actually fix a bug that the map is not moving at all for example? 🤔